### DISCUSSION TITLE
Lesson 3 Error: not found artifacts/SimpleStorage.sol

### DISCUSSION THREAD

**Discussion by SNERUOL** [Cyfrin/foundry-full-course-cu#1194] [Q&A] [answered]
Could somebody help me with this error message please?

I followed along and as you can see in the screenshot I do actually have a SimpleStorage.sol file created in the previous section of lesson 3. Still I got this error message 

![Schermafbeelding 2023-12-11 220821](https://github.com/Cyfrin/foundry-full-course-f23/assets/153216108/39e5c06e-247a-4853-a8fd-cc336a0a56ff)
![Schermafbeelding 2023-12-11 220808](https://github.com/Cyfrin/foundry-full-course-f23/assets/153216108/dd72d557-1924-4f1e-8e94-d0d8b1f98536)
![Schermafbeelding 2023-12-11 220735](https://github.com/Cyfrin/foundry-full-course-f23/assets/153216108/ff4dbcb0-457d-4d64-b6f6-cd2994be7149)

**Comment by pratokko** [comment]
Hello, not found probably means that the file cannot be found make sure your file is named SimpleStorage.sol

**Reply by SNERUOL** [reply]
Hi pratokko, thanks for your reply. I guess my spelling is good right, could you please confirm from my screenshot?

**Reply by pratokko** [reply, marked as answer]
Are you creating the file in the artifacts folder? check to ensure it is not in the artifacts folder

**Reply by SNERUOL** [reply]
Thanks! i saw it too at a later moment and was too busy with the course that I forgot to reply in here. But thanks for your help!
